### DISCUSSION TITLE
How to use drops?

### DISCUSSION THREAD

**Discussion by espen** [harttle/liquidjs#591] [Q&A]
Sorry if I missed this but I looked through the website and searched the repo and couldn't find any documentation on how to use drops. I use drops with Liquid Ruby and looking into how I can create and use a similar drop with LiquidJS.

Are there any drop usage examples? It would be helpful to see it even if there is no documentation on this.

**Comment by harttle** [comment]
Hi @espen, here're some test cases https://github.com/harttle/liquidjs/blob/1380ac931a51dcb236c913aba64acc613f81c1ad/test/integration/drop/drop.ts maybe useful to you.

Thanks for the inquery, there's no docs on Drop yet,  definitely we need at least one introduction doc for it. Just created an issue #592 to track.
